src/aco.cpp

```diff
@@ -75,11 +75,21 @@ namespace full_coverage_path_planner
 
         Point_t start = {init.x, init.y};
 
+        int velocity = 5;
         for (int i = 0; i < 1000; ++i)
         {
+            if(i > 200 && i < 400)
+                velocity = 4;
+            else if(i >= 400 && i < 500)
+                velocity = 3;
+            else if(i >= 500 && i < 700)
+                velocity = 2;
+            else
+                velocity = 1;
+            
```

Comment:
Ant construction loop.

In here, the varying velocity of the ants is specified as follows:
0-200 (200 total): 5tpi (tiles per iteration)
200-400 (200 total): 4tpi
400-500 (100 total): 3tpi
500-700 (200 total): 2tpi
700-1000 (300 total): 1tpi

These numbers are guesswork and could definitely use some improvement.